foundry.toml

```diff
@@ -27,6 +27,7 @@ max_test_rejects = 128_000
 [etherscan]
   mainnet = { key = "${API_KEY_ETHERSCAN}" }
   sepolia = { key = "${API_KEY_ETHERSCAN}" }
+  2442 = { key = "${API_KEY_CARDONA}", url = "https://api-cardona-zkevm.polygonscan.com/api" }
```

Comment:
Curious: why 2442?

Reply:
because cardona (the new version of the polygon zkevm testnet) is not supported by name yet in `forge`, hence, we must use the chain id instead.